diesel/Cargo.toml

```diff
@@ -33,13 +34,12 @@ bitflags = { version = "0.9", optional = true }
 
 [dev-dependencies]
 cfg-if = "0.1.0"
-diesel_codegen = "0.16.0"
 dotenv = ">=0.8, <0.11"
 quickcheck = "0.3.1"
 tempdir = "^0.3.4"
 
 [features]
-default = ["with-deprecated"]
+default = ["with-deprecated", "sqlite"]
```

Comment:
Wait, why are we including sqlite by default now?

Reply:
Not really (It is only a artifact from testing…)